### PR TITLE
Add new study: Collette_2021

### PR DESCRIPTION
**Pull request by ehwenk**
* add large study looking at seed dormancy
* data from online repository; authors have not yet responded to email

### CODE REVIEW

File: data/Collette_2021/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+  find: PD
+  replace: physiological_dormancy
+- trait_name: seed_dormancy_class
+  find: PY
+  replace: physical_dormancy
+- trait_name: seed_dormancy_class
+  find: MPD
+  replace: morphophysiological_dormancy
+- trait_name: seed_dormancy_class
+  find: MD
```
Comment (yangsophieee) [inline data/Collette_2021/metadata.yml:154]: It doesn't seem like they scored a single species as MD (morphological dormancy)

File: data/Collette_2021/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+    was determined by a combination of author knowledge, seed morphological characteristics
+    and information from germination trials conducted by Botanic Gardens' researchers
+    when viability testing seed collections. It was not possible to assign dormancy
+    class confidently to a species level across the full dataset, due to a lack of
+    available literature on every species. As such, some genera that were subject
+    to conflicting reports, where we were not confident in assigning dormancy class,
+    were excluded from the analysis.
+substitutions:
+- trait_name: seed_dormancy_class
+  find: ND
```
Comment (yangsophieee) [inline data/Collette_2021/metadata.yml:142]: I didn't know that capitalisation doesn't matter for substitutions! The data is all lowercase.

**Review by yangsophieee — APPROVED** [review]
What a large dataset!

File: data/Collette_2021/metadata.yml [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+      of rainfall seasonality and temperature'
+    volume: '127'
+    number: '5'
+    pages: 613--620
+    doi: 10.1093/aob/mcaa203
+contributors:
+  data_collectors:
+  - last_name: Collette
+    given_name: Justin
+    ORCID: 000-0002-4618-2677
```
[the remark this answers is not in the document]
Comment (yangsophieee) [reply data/Collette_2021/metadata.yml]: I've fixed it.

[the remark this answers is not in the document]
Comment (ehwenk) [reply data/Collette_2021/metadata.yml]: Yes, good idea.

File: data/Collette_2021/metadata.yml [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+    affiliation: Centre for Ecosystem Science, School of Biological, Earth, and Environmental
```
[the remark this answers is not in the document]
Comment (ehwenk) [reply data/Collette_2021/metadata.yml:19]: I could also ask Mark... Justin hasn't gotten back to my emails yet.

File: data/Collette_2021/metadata.yml [hunk 5]
```diff
@@ -0,0 +1,348 @@
...
+    District Council of Lower Eyre Peninsula for South Australia, (3) for NSW, the
+    'Central Coast' botanical sub-division from the east coast of New South Wales
+    and (4) for TAS, a customized polygon outlining the small aseasonal rainfall region
+    of the north-east coast of Tasmania (Fig. 1). Species lists from each of our defined
+    regions were obtained and then manually filtered to include only native shrubs.
+    There were similar proportions of species present from the same families across
+    all four study regions, with just six families (Fabaceae, Proteaceae, Myrtaceae,
+    Ericaceae, Rutaceae and Asteraceae) making up almost 70 % of species in all regions
+    ( Appendix 1).
+  original_file: none; sourced from Dryad
```
[the remark this answers is not in the document]
Comment (ehwenk) [reply data/Collette_2021/metadata.yml:76]: no, I just wrote the wrong thing

File: data/Collette_2021/metadata.yml (hunk 1 above)
Comment (ehwenk) [reply data/Collette_2021/metadata.yml:154]: I noticed that too, but decided to leave it - just for completeness.

File: data/Collette_2021/metadata.yml (hunk 2 above)
Comment (ehwenk) [reply data/Collette_2021/metadata.yml:142]: yes - everything turns lowercase for matching.

**Pull request merged**